runtime/vam/expr/function/errors.go

```diff
@@ -0,0 +1,34 @@
+package function
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/vector"
+)
+
+// https://github.com/brimdata/zed/blob/main/docs/language/functions.md#quiet
+type Quiet struct {
+	zctx *zed.Context
+}
+
+func (q *Quiet) Call(args ...vector.Any) vector.Any {
+	arg, ok := args[0].(*vector.Error)
+	if !ok {
+		return args[0]
+	}
+	if _, ok := arg.Typ.Type.(*zed.TypeOfString); !ok {
+		return args[0]
+	}
+	n := arg.Len()
+	vec := vector.NewStringEmpty(n, vector.NewBoolEmpty(n, nil))
+	for i := uint32(0); i < n; i++ {
+		s, isnull := vector.StringValue(arg.Vals, i)
+		if isnull {
+			vec.Nulls.Set(i)
+		}
```

Comment:
Nit: Just use arg.Nulls.
```suggestion
	vec := vector.NewStringEmpty(n, arg.Nulls)
	for i := uint32(0); i < n; i++ {
		s, _ := vector.StringValue(arg.Vals, i)
```